src/background.js

```diff
@@ -271,18 +278,26 @@ function setLibraryDialog() {
 	})
 }
 
-async function getCurrentInterfaceJSON() {
-    var interfacePackageJSON = storagePath.interfaceSettings + '/interface_package.json';
-    let rawdata = fs.readFileSync(interfacePackageJSON);
-    let interfaceJSON = JSON.parse(rawdata);
-    
-    if (interfaceJSON) {
-        console.info("Interface Package JSON:", interfaceJSON);
-        return interfaceJSON;
-    } else {
-        return false;
-    }
-}
+// async function getCurrentInterfaceJSON() {
```

Comment:
Remove commented-out code?

Reply:
This will be used again in the future most likely.